clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -329,7 +328,8 @@ public class VenicePushJob implements AutoCloseable {
    */
   public static final String REPUSH_TTL_IN_HOURS = "repush.ttl.hours";
   public static final String REPUSH_TTL_POLICY = "repush.ttl.policy";
```

Comment:
I know we supports multiple TTL policies, and I am wondering whether it is necessary to apply TTL to batch data or not, and in theory, TTL repush can only be applied to RT write since for batch-part, we don't know the true write time of the batch data.

I am trying to see how we could simplify TTL repush.

For a real-time only store, I assume TTL repush could be the default mode for repush and no extra config is required, and the TTL will be the rewind time configured in store.

Reply:
I know it seems a bit over-engineering but I want to explicitly state that we want to be flexible here in case in the future we are able to know the true write time of the batch data :wink:.

That being said, the `real-time only` is the only policy we support and set as default. Customers won't be able to change it until we support more.

> For a real-time only store, I assume TTL repush could be the default mode for repush and no extra config is required, and the TTL will be the rewind time configured in store.

I am not sure I understand but the TTL repush is not the default mode for real-time only store repush. It will be turn on if and only if `repush.ttl.hours` is specified.

Reply:
Have made this feature a flag and use the store-level rewind time